inputs/values/ref_panel_1kg.json

```diff
@@ -1168,6 +1168,7 @@
     ],
     "clean_vcf": "gs://gatk-sv-ref-panel-1kg/outputs/MakeCohortVcf/8a209488-c928-449d-92cd-0a5131e92b7c/call-CleanVcf/CleanVcf/277f3f25-bb99-4fe4-a48b-567fd3f344f9/call-ConcatCleanedVcfs/ref_panel_1kg.cleaned.vcf.gz",
     "clean_vcf_index": "gs://gatk-sv-ref-panel-1kg/outputs/MakeCohortVcf/8a209488-c928-449d-92cd-0a5131e92b7c/call-CleanVcf/CleanVcf/277f3f25-bb99-4fe4-a48b-567fd3f344f9/call-ConcatCleanedVcfs/ref_panel_1kg.cleaned.vcf.gz.tbi",
+    "clean_vcf_gatk_formatter_args": "--use-end2",
```

Comment:
I believe this clean VCF is from later than v0.22-beta, so based on your filtering Google Doc should this be `""` instead?

Reply:
Yes that's a mistake in the doc, which I've now updated. I've also changed this argument to the opposite `--fix-end` which is a little clearer and not required for current VCFs.